pyanaconda/ui/webui/__init__.py

```diff
@@ -109,6 +110,10 @@ def run(self):
                 "/usr/bin/systemctl", "enable", "--now", "cockpit.socket"
             ])
 
+        # Force Firefox to be used via the BROWSER environment variable.
+        # This is read by cockpit-desktop and makes it launch Firefox in kiosk mode
+        # instead of the GTK WebKit based web view it launches by default.
+        os.environ["BROWSER"] = "firefox --kiosk"
```

Comment:
Just shut up pylint like so:
```suggestion
        os.environ["BROWSER"] = "firefox --kiosk"  # pylint: disable=environment-modify
```

(Sometimes I wonder why that rule is on... but we do have only a few places where changing environment makes sense, so it's probably correct.)